frontend/src/views/components/Dashboard/GraphGrid/GraphGrid.tsx

```diff
@@ -1,47 +1,45 @@
-import Grid from "@mui/material/Grid2";
-import { Container, Card, CardContent, CardHeader } from "@mui/material";
+// src/components/Dashboard/GraphGrid/GraphGrid.tsx
+import { Container, Grid, Card, CardContent, CardHeader } from "@mui/material";
 import BarChart from "../BarChart/BarChart";
+import InfoButton from "../InfoButton/InfoButton"; // Import the InfoButton component
 
-type BarChartData = {
-    label: string;
-    value: number;
-};
-
-interface GraphProps {
-    data: BarChartData[];
-    title: string;
-}
-
-function Graph({ data, title }: GraphProps) {
-    return (
-        <Card sx={{ width: 325, p: 3 }}>
-            <CardHeader title={title} />
-            <CardContent>
-                <BarChart data={data} />
-            </CardContent>
-        </Card>
-    );
+interface GraphGridProps {
+  graphsInfo: { values: number[] }[]; // Existing structure of graphsInfo
```

Comment:
i like the use of the interface! Just a quick question, do we no longer need a label string? as you removed the BarChartData type